code/index.html

```diff
@@ -23,25 +23,56 @@
       <h1>Does the person have</h1>
 
       <select id="questions">
+          <!-- Options of hair colors  -->
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
+          <option value="purple">purple hair</option>
+          <option value="black">black hair</option>
+          <option value="orange">orange hair</option>
+          <option value="grey">grey hair</option>
+          <option value="white">white hair</option>
+          <option value="hidden">no hair </option>
         </optgroup>
+          <!-- Options of eye colors  -->
         <optgroup label="eye color">
           <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
+          <option value="blue">blue eyes</option>
+          <option value="grey">grey eyes</option>
+          <option value="green">green eyes</option>
+          <option value="hazel">hazel eyes</option>
         </optgroup>
+          <!-- Options accessories  -->
         <optgroup label="accessories">
           <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
+          <option value="headgear">headgear</option>
+          <option value="necklace">necklace</option>
+          <option value="earrings">earrings</option>
+          <option value="patterns">patterned shirt</option>
         </optgroup>
+        <!-- Options of other traits  -->
         <optgroup label="other">
-          <option value="smoker">a smoking habit</option>
+          <option value="smokinghabit">a smoking habit</option>
+          <option value="phone">mobile phone</option>
+          <option value="beard">facial hair</option>
+          <option value="button">buttons</option>
         </optgroup>
       </select>
 
-      <button id="filter" class="filled-button">FIND OUT</button>
+      <button id="filter" class="filled-button">FIND OUT</button><br>
+
+  <div class="counter">
+  <span>Questions asked:</span>
+  <span id="totalGuesses" class="totalGuesses"></span>
```

Comment:
This element (and on line 71) has a `class="totalGuesses"` that is not used in the stylesheet or the script file. You should therefore remove it. 
Also classes should be written in kebab-case, like this: `class="total-guesses" `

Reply:
Yeah, rest of struggles to make the counters look like buttons. Thank you!